source/brailleDisplayDrivers/eurobraille/driver.py

```diff
@@ -287,7 +287,7 @@ def _handleKeyPacket(self, group: bytes, data: bytes):
 			log.debug("Ignoring key repetition")
 			return
 		self.keysDown[group] |= arg
-		isIris = self.deviceType.startswith("Iris")
+		isIris = self.ProtocolType.startswith("Iris")
```

Comment:
shouldn't this use `deviceType` too?

Reply:
Yes, turned out that I created a bad find/replace.
```suggestion
		isIris = self.deviceType.startswith("Iris")
```